app/Http/Controllers/AdsController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\AdsRequest;
+use App\Http\Resources\AdsResource;
+use App\Models\Page;
+use Illuminate\Http\Request;
+
+class AdsController extends Controller
+{
+    /**
+     * Get ads list filtered by the specific page 
+     * 
+     * @param AdRequest $request
+     * @param Page $pageName
```

Comment:
Doesn't looks correct. There is only one param. Probably you need to use Swagger documentation. Please check other Controllers